pyproject.toml

```diff
@@ -70,6 +70,8 @@ flake8 = "*"
 bumpversion = "*"
 black = "23.*"
 isort = "*"
+schemathesis = "*"
+starlette = ">=0.13,<0.21"
```

Comment:
It would be nice to refer here to the schemathesis bug that necessitated this

Reply:
Installing (pinned) Starlette is not required when pinning Schemathesis version to `3.18.*`, which is probably a good idea because Schemathesis is being developed quite rapidly.